analyzers/tests/SonarAnalyzer.Test/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.vb

```diff
@@ -1711,12 +1711,12 @@ Public Class NullOrEmpty
 
         'Combo/Fatality
         ret = If(isNotNothing, isNothing)
-        '        ^^^^^^^^^^^^                             Noncompliant {{Change this expression which always evaluates to the same result. Some code paths are unreachable.}}
+        '        ^^^^^^^^^^^^                             Noncompliant {{Remove this unnecessary check for null. Some code paths are unreachable.}}
```

Comment:
Not sure about VB.NET: is `check for null` good enough or should it be `check for Nothing`?

Reply:
It should be `Nothing`.